CMakeLists.txt

```diff
@@ -5,6 +5,15 @@ project(box2d VERSION 2.4.0)
 
 # set(CMAKE_CONFIGURATION_TYPES "Debug;RelWithDebInfo" CACHE STRING "" FORCE)
 
+include(GNUInstallDirs)
+
+if(NOT GENERATOR_IS_MULTI_CONFIG)
```

Comment:
Unfortunately, this is not a variable, it's a property. So you have to use something like that:
`get_property(_isMultiConfig GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)`

and then use `_isMultiConfig` in the condition.